discord/file.py

```diff
@@ -60,8 +60,10 @@ class File:
         The filename to display when uploading to Discord.
         If this is not given then it defaults to ``fp.name`` or if ``fp`` is
         a string then the ``filename`` will default to the string given.
-    spoiler: :class:`bool`
+    spoiler: Optional[:class:`bool`]
         Whether the attachment is a spoiler.
+    description: Optional[:class:`str`:]
+        The file description can be set to attached Images to show a alternative Text.
```

Comment:
Nouns are written in lower case in English (with the exclusion of proper nouns). 😉